src/pages/Studio/index.js

```diff
@@ -56,16 +58,14 @@ export default ({ location, isDesktop }) => {
   }, [])
 
   useEffect(() => {
-    if (parsedUrl && parsedUrl.layout) {
+    if (layout && layout.id) {
+      mutateStoreUserActivity(EntityKeys.CHART_CONFIGURATION, layout.id, InteractionType.VIEW)
```

Comment:
@peratik Not sure if it will work for anonymous user

Reply:
@DmitriVanGuard  useUser and isLoggedIn added